core/src/main/java/com/google/instrumentation/stats/RpcConstantsFromProtoBinary.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2016, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.stats;
+
+import com.google.common.io.Files;
```

Comment:
Sorry for the confusion, this PR is intended to demonstrate one way to read pre-defined constants from protobuf binaries. With this approach we need to add guava and proto as dependency to stats_core. However after talking with Dino, we find this is not a desirable approach, because for some reason stats_core cannot directly depend on proto. I'll try not to add additional dependencies to stats_core and update this PR later.